pages/common/loc.md

```diff
@@ -0,0 +1,19 @@
+# loc
+
+> Tool written in Rust that counts lines of code quickly.
```

Comment:
We try to be objective in our descriptions. Let us remove "quickly" as it does not add any new info.